pandas/core/aggregation.py

```diff
@@ -608,92 +608,41 @@ def aggregate(obj, arg: AggFuncType, *args, **kwargs):
 
         from pandas.core.reshape.concat import concat
 
-        def _agg_1dim(name, how, subset=None):
-            """
-            aggregate a 1-dim with how
-            """
-            colg = obj._gotitem(name, ndim=1, subset=subset)
-            if colg.ndim != 1:
-                raise SpecificationError(
```

Comment:
i take it this is never reachable?  IIUC thats the lynchpin to the cleanup

Reply:
That's right; this is only ever called via `_agg`, so `name` here is a key in `arg`. It must therefore be a single label.